app/models/user.rb

```diff
@@ -58,6 +58,16 @@ def authenticated?(attribute, token)
     BCrypt::Password.new(digest).is_password?(token)
   end
 
+  def suspend
+    self.suspended = true
+    self.save
+  end
+
+  def unsuspend
+    self.suspended = false
+    self.save
+  end
```

Comment:
I think you can leave out the self and it'l just be implied. Something to think about in future refactors but this works!